assets/src/dashboard/components/alert/components.js

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import styled, { keyframes } from 'styled-components';
+
+/**
+ * Internal dependencies
+ */
+import {
+  ALERT_SEVERITIES,
+  KEYBOARD_USER_SELECTOR,
+  Z_INDEX,
+} from '../../constants';
+import { TypographyPresets } from '../typography';
+
+const slideIn = keyframes`
+	from {
+		transform: translateX(100%);
+	}
+	to {
+		transform: translateX(0);
+	}
+`;
+
+const alertBackgrounds = {
+  [ALERT_SEVERITIES.ERROR]: 'danger',
+  [ALERT_SEVERITIES.WARNING]: 'warning',
+  [ALERT_SEVERITIES.INFO]: 'bluePrimary',
+  [ALERT_SEVERITIES.SUCCESS]: 'success',
+  [ALERT_SEVERITIES.DEFAULT]: 'bluePrimary',
+};
+
+const getColor = (severity = ALERT_SEVERITIES.DEFAULT) => {
+  return alertBackgrounds[severity];
+};
+
+export const Wrapper = styled.div`
+  position: fixed;
+  bottom: 40px;
+  right: 0;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  max-width: 300px;
+  min-width: 40vw;
+`;
+
+export const AlertContainer = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  padding: 10px 20px;
+  margin-top: 20px;
+  color: ${({ theme }) => theme.colors.white};
+  background-color: ${({ theme, severity }) =>
+    theme.colors[getColor(severity)]};
+  border-radius: 5px;
+  z-index: ${Z_INDEX.ALERT};
+  animation: 0.5s ${slideIn} ease-in;
+`;
+
+export const AlertText = styled.p`
+  ${TypographyPresets.Medium};
+  width: calc(100% - 25px);
+  display: inline;
+`;
+
+export const DismissButton = styled.button`
+  align-self: center;
+  margin: 0 0 0 auto;
+  width: 25px;
+  height: 25px;
+  background-color: transparent;
+  color: ${({ theme }) => theme.colors.white};
+  border: ${({ theme }) => theme.borders.transparent};
+  cursor: pointer;
```

Comment:
isn't this part of the default styling for the button? was it behaving differently without it?

Reply:
i opted to not use the component `Button` here because i'd have to overwrite so much of it. The default element `button` doesn't have the `pointer` on hover by default though so I added it here for consistency with the Button component.

Reply:
Ah yea guess we stripped this default with our base styling. good to know. thanks!